### PR TITLE
Render empty state on the Unmint page

### PR DESCRIPTION
**Pull request by r-czajkowski**
Render an empty state on the Unmint page. Here we also extract empty state component to a separate file since we want to render the same empty state for unmint and mint page.

<details>
  <summary>Screenshots</summary>
  
obraz
  
</details>

### CODE REVIEW

File: src/pages/tBTC/Bridge/Unmint.tsx [hunk 1]
```diff
@@ -286,7 +289,22 @@ UnmintFormPage.route = {
...
-  return <Outlet />
+  const { active } = useWeb3React()
+
+  return (
+    <BridgeLayout>
+      <BridgeLayoutMainSection>
+        {active ? (
+          <Outlet />
+        ) : (
+          <BridgeProcessEmptyState title="Ready to unmit tBTC?" />
```
Comment (michalsmiarowski) [inline src/pages/tBTC/Bridge/Unmint.tsx:300]: After connecting the wallet while being on redemption page i see something like this:

image

I believe this is not expected behaviour

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]
Found bug in UI, see comment below

File: src/pages/tBTC/Bridge/Unmint.tsx (hunk 1 above)
Comment (r-czajkowski) [reply src/pages/tBTC/Bridge/Unmint.tsx]: f35c091

**Review by michalsmiarowski — APPROVED** [review]
LGTM 🖖

**Pull request merged**